apps/test/unit/lib/ui/accounts/ChangeEmailFormTest.jsx

```diff
@@ -230,11 +231,12 @@ describe('ChangeEmailForm', () => {
     it('the opt-in field is disabled', () => {
       wrapper.setProps({userType: 'teacher'});
       expect(wrapper.find(OPT_IN_SELECTOR)).to.have.attr('disabled');
+      expect(wrapper.find(OPT_OUT_SELECTOR)).to.have.attr('disabled');
     });
   });
 
   describe('focusOnAnError()', () => {
-    let wrapper, emailSpy, passwordSpy, optInSpy;
```

Comment:
I think this was testing focusing on the dropdown, which wasn't really happening in the component and doesn't make as much sense for radio buttons, so I removed it. Happy to add it back if I'm wrong